crates/aptos/src/node/mod.rs

```diff
@@ -1526,6 +1532,49 @@ impl CliCommand<()> for BootstrapDbFromBackup {
     }
 }
 
+/// Checks the network connectivity of a node by dialing the node using noise
+#[derive(Parser)]
+pub struct CheckNetworkConnectivity {
+    /// `NetworkAddress` of remote server interface.
+    /// Examples include:
+    /// - `/dns/example.com/tcp/6180/noise-ik/<x25519-pubkey>/handshake/1`
+    /// - `/ip4/<ip-address>/tcp/6182/noise-ik/<x25519-pubkey>/handshake/0`
+    #[clap(long, value_parser = validate_address)]
+    pub address: NetworkAddress,
+
+    /// `ChainId` of remote server.
+    /// Examples include: `devnet`, `testnet`, `mainnet`.
```

Comment:
Maybe a comment on how you have to use `testing` to test your local testnet, if that's what you're doing.

Reply:
Please use ChainId.  The FromStr should be able to take MAINNET and 25 in the same space, rather than using the named chain modifications

Reply:
Sweet, thanks folks! 😄
1. Added `testing` and a comment about local test networks.
2. Changed it back to `chain_id` and tested that chain IDs and chain names worked